yeps/yep-111.md

```diff
@@ -23,97 +23,110 @@ Fields are inspired by Bluesky, which has a system of [fields](http://nsls-ii.gi
 For example, a simple motor might have the field "position" and the field "velocity".
 Users might expect a graphical client to display the current value of each of these fields and allow setting their value, where possible.
+We will use the word "properties" to match the conventions of the Python ecosystem.
 
 Using Avro-RPC (see [YEP-107](https://yeps.yaq.fyi/107)), yaq daemons completely specify their client interface.
 A client knows the exact signature of every message that the daemon supports.
 In addition, yaq has a traits system which defines preferred message syntax for common behaviors.
 Client developers may use the traits system to safely build user interfaces.
-Despite these tools, the relationship between message protocols and logical "fields" of the daemon is often not obvious.
-Because individual daemons are allowed to specify arbitrary messages beyond those implied by traits, it can be a challenge to automatically discover the relevant fields for an arbitrary daemon.
+Despite these tools, the relationship between message protocols and logical "properties" of the daemon is often not obvious.
+Because individual daemons are allowed to specify arbitrary messages beyond those implied by traits, it can be a challenge to automatically discover the relevant properties for an arbitrary daemon.
 
 A typical graphical client needs to know the following:
 
-- what informational fields should be displayed?
-- which fields can be set, and which are read-only?
-- what messages are used to read and write a given field?
-- which read-only fields are dynamic?
-- which fields are primary and which are secondary?
+- what informational properties should be displayed?
+- which properties can be set, and which are read-only?
+- what messages are used to read and write a given property?
+- which read-only properties are dynamic?
+- which properties are especially important to most users?
 
-Here "field" is a single piece of information about a daemon, things like "current position", "integration time", and "averaging method".
-This YEP allows daemons to help connected clients by specifying relevant fields in a standardized way.
-While the primary purpose of fields is automatic user-interface generation, fields may also be useful when choosing what (meta)data to record from each daemon in the context of an experiment.
+Here "property" is a single piece of information about a daemon, things like "current position", "integration time", and "averaging method".
+This YEP allows daemons to help connected clients by specifying relevant properties in a standardized way.
+While the primary purpose of properties is automatic user-interface generation, fields may also be useful when choosing what (meta)data to record from each daemon in the context of an experiment.
 
 Fields exist on-top-of and do not modify or define the Avro-RPC protocol itself.
 Clients may ignore fields without penalty.
```

Comment:
```suggestion
Clients may ignore properties without penalty.
```

Reply:
What does it mean to ignore here?

Reply:
Thanks for the question @kameyer226 

I'm trying to point out that properties are fully optional. A client NEEDS to pay attention to the messages, as their description is needed to encode commands. Simple clients can choose to not use the additional metadata of properties.